commands/miner.go

```diff
@@ -450,3 +451,37 @@ Values will be output as a ratio where the first number is the miner power and s
 		}),
 	},
 }
+
+var minerCollateralCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline:          "Get the active collateral of a miner",
+		ShortDescription: `Check the actively staked collateral of a given miner. Values reported in attoFIL`,
+	},
+	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
+		minerAddr, err := optionalAddr(req.Arguments[0])
+		if err != nil {
+			return err
+		}
+
+		bytes, err := GetPorcelainAPI(env).MessageQuery(
```

Comment:
`bytes` is a misleading name for `[][]byte`. The miner porcelain code calls this `rets`.